migrations/52-add-error-reason.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE transaction ADD COLUMN status_reason varchar(255);
```

Comment:
varchar columns are nullable, so mysql makes them null by default, I can add to be explicit though :)